go/api/request_routing_config.go

```diff
@@ -111,6 +121,10 @@ func (slotIdRoute *SlotIdRoute) toRoutesProtobuf() (*protobuf.Routes, error) {
 	return request, nil
 }
 
+func (route *SlotIdRoute) isMultiNode() bool {
```

Comment:
Should we consolidate isMultiNode() false implementations into a single struct and then compose that struct into `SlotIdRoute`, `SlotKeyRoute` and `ByAddressRoute` structs? This would eliminate some code duplication and make it clear which routes don't support MultiNode.

```go
type notMultiNode struct {}

func (r *notMultiNode) isMultiNode() bool { return false }

type SlotIdRoute struct {
    slotType SlotType
    slotID int32
    notMultiNode
}
```

Reply:
Wow, that works amazing!